geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/DestroyDataSourceCommand.java

```diff
@@ -74,6 +77,15 @@ public ResultModel destroyDataSource(
             "Data source named \"{0}\" does not exist. A jndi-binding was found with that name.",
             dataSourceName));
       }
+
+      try {
+        checkIfDataSourceIsInUse(service, dataSourceName);
+      } catch (IllegalStateException ex) {
+        return ResultModel.createError(CliStrings.format(
+            "Data source named \"{0}\" is still being used by region \"{1}\". Use destroy jdbc-mapping --name={1} and then try again.",
+            dataSourceName, ex.getMessage()));
```

Comment:
Doesn't look like ex.getMessage() is adding any helpful info here.

Reply:
it is adding the name of the region/jdbc-mapping that is using the data source